boards/adafruit-clue/Makefile.dep

```diff
@@ -0,0 +1,21 @@
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += saul_gpio
+  USEMODULE += apds9960
+  USEMODULE += bmp280_i2c
+  USEMODULE += lis3mdl
+  USEMODULE += sht3x
+endif
+
+# Use stdio over USB
+DEFAULT_MODULE += stdio_cdc_acm
+
+ifneq (,$(filter stdio_cdc_acm,$(USEMODULE)))
+  # This boards requires support for nrfutil bootloader.
+  FEATURES_REQUIRED += bootloader_nrfutil
+
+  # and for the board reset via USB mechanism
+  USEMODULE += usb_board_reset
+endif
```

Comment:
This will need including the `stdio` blacklisting code before. Maybe its worth splitting that handling out from `stdio.in.mk`, so it can be included when needed earlier, like in this case. Otherwise you could change it to this and address later. But since its used here and I also use it #13664, I think its worth consolidating.

```suggestion
ifneq (,$(filter-out stdio_cdc_acm,$(filter stdio_%,$(USEMODULE))))
  DISABLE_MODULE += stdio_cdc_acm
endif

ifneq (,$(filter stdio_cdc_acm,$(USEMODULE)))
  # This boards requires support for nrfutil bootloader.
  FEATURES_REQUIRED += bootloader_nrfutil

  # and for the board reset via USB mechanism
  USEMODULE += usb_board_reset
endif
```